js/LocationButtons.js

```diff
@@ -0,0 +1,25 @@
+export default class LocationButtons extends HTMLElement {
+    constructor() {
+        super();
+        this.locationValue;
```

Comment:
Grundsätzlich musst du eine Class Property nicht initialisieren. Was du hier machst, ist im Wesentlichen: `this.locationValue = undefined`. Wenn du an einem anderen Ort `this.locationValue` abfragst, wird dir JavaScript mit oder ohne diese Initialisierung `undefined` zurückgeben, weil jede Property, die nicht gesetzt ist, undefined ist.

Reply:
Ah, wenn wenn initialisieren, dann gern mit `null`. Dann weisst du: Ah, da hta mal wer was gesetzt und absichtlich noch nicht mit einem Wert versehen.